framework/physics/FieldFunction/fieldfunction_gridbased.h

```diff
@@ -67,36 +58,53 @@ class FieldFunctionGridBased : public FieldFunction
 
   virtual ~FieldFunctionGridBased() = default;
 
-private:
-  VectorGhostCommPtr MakeGhostCommunicator();
-
-public:
   // Getters
+  /**Returns the spatial discretization method.*/
   const chi_math::SpatialDiscretization& SDM() const { return *sdm_; }
```

Comment:
```suggestion
  /**Returns the spatial discretization.*/
  const chi_math::SpatialDiscretization& SpatialDiscretization() const { return *sdm_; }
```
Function name and doc is a little bit misleading. Ideally `sdm_` -> `discretization_`.

Reply:
Can't use `SpatialDiscretization` but did use `GetSpatialDiscretization`